gh-actions/common/build-debian/action.yml

```
@@ -51,10 +55,20 @@ runs:
         DEBIAN_FRONTEND=noninteractive sudo apt install -y devscripts
         echo "::endgroup::"
 
-        echo "::group::Append commit SHA to local version"
+        echo "::group::Create local version with commit and docker container"
         cd '${{ inputs.source-dir }}'
-        sanitized_docker=$( echo "${{ inputs.docker-image }}" | sed 's/://' )
-        debchange --local "+${sanitized_docker}+${{ github.sha }}" "Github build. Job id: ${{ github.run_id }}. Attempt: ${{ github.run_number }}."
+
+        # Sanitize the docker name so that it stick to debian policy
+        # https://www.debian.org/doc/debian-policy/ch-controlfields.html#version
+        sanitized_docker=$( echo "${{ inputs.docker-image }}" | sed -r 's/[^a-zA-Z0-9.+~]+/+/g' )
+
+        # Short commit to avoid "package-has-long-file-name"
+        commit=$(echo ${{ github.sha }} | cut -c1-8)
+
+        export DEBFULLNAME="GitHub actions runner"
+        export DEBEMAIL="noreply@github.com"
```

Or maybe latest commit author?

```suggestion
        export DEBFULLNAME="$(git log -1 --format='%an' HEAD) - GH Action"
        export DEBEMAIL="$(git log -1 --format='%ae' HEAD)"
```

Plus an indication that is GH though, to make clear both things.